packages/neuron-ui/src/components/Receive/index.tsx

```diff
@@ -31,7 +31,8 @@ const Receive = ({
   const [showLargeQRCode, setShowLargeQRCode] = useState(false)
 
   const accountAddress = useMemo(
-    () => params.address || (addresses.find(addr => addr.type === 0) || { address: '' }).address || '',
+    () =>
+      params.address || (addresses.find(addr => addr.type === 0 && addr.txCount > 0) || { address: '' }).address || '',
```

Comment:
```suggestion
      params.address || (addresses.find(addr => addr.type === 0 && addr.txCount === 0) || { address: '' }).address || '',
```

We're looking for unused address.